lcls_tools/common/model/gaussian.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+from scipy.stats import norm, gamma
+import lcls_tools.common.model.optimize as optimize
+
+
+def curve(x, mean=0, sigma=1, amp=1, off=0):
+    A = np.sqrt(2 * np.pi) * amp
+    return A * norm.pdf((x - mean) / sigma) + off
+
+
+class mean(optimize.Parameter):
+    name = "mean"
+    bounds = (0, 1)
+
+    @staticmethod
+    def init(x, y):
+        return np.average(x, weights=y)
+
+    @staticmethod
+    def prior(mean, mean_0):
+        return norm(mean_0, 0.1).logpdf(mean)
+
+    @staticmethod
+    def denormalize(mean, x, y):
+        x_scale = np.max(x) - np.min(x)
+        return mean * x_scale + min(x)
+
+
+class sigma(optimize.Parameter):
+    name = "sigma"
+    bounds = (1e-10, None)
+
+    @staticmethod
+    def init(x, y):
+        return np.sqrt(np.cov(x, aweights=y))
+
+    @staticmethod
+    def prior(sigma, sigma_0):
+        return gamma(2.5, loc=0, scale=1 / 5.0).logpdf(sigma)
+
+    @staticmethod
+    def denormalize(sigma, x, y):
+        x_scale = np.max(x) - np.min(x)
```

Comment:
Is normalization different for sigma? This doesn't match the equation above? Also, same question about the `y` here.